GitUI/Editor/Diff/DiffLineNumAnalyzer.cs

```diff
@@ -70,20 +77,24 @@ public DiffLinesInfo Analyze(string diffContent, bool isCombinedDiff)
 
                 ret.Add(meta);
             }
-            else if (isHeaderLineLocated && IsMinusLine(line))
+            else if (isHeaderLineLocated && ((!isGitWordDiff && IsMinusLine(line))
+
+                // Heuretics: For GitWordDiff AppSettings.UseGEThemeGitColoring is assumed, otherwise just DiffLineType.Mixed is detected
```

Comment:
```suggestion
                //// Heuristics: For GitWordDiff, AppSettings.UseGEThemeGitColoring is assumed, otherwise just DiffLineType.Mixed is detected
```

Reply:
Why ////

Sidenote: If AppSettings.UseGEThemeGitColoring is not used, dominant red/green could be used. But as Mixed is the fallback, I do not bother for that special case.

Reply:
> Why ////

In order to get rid of that irritating empty line.

Reply:
It is a consequence of the style applied, live with it.
Dont cheat to get around the rules and keep the rules documented if they cannot be enforced